Neos.Neos/Classes/PendingChangesProjection/ChangeProjection.php

```diff
@@ -183,7 +184,7 @@ public function apply(EventInterface $event, EventEnvelope $eventEnvelope): void
     {
         match ($event::class) {
             RootWorkspaceWasCreated::class => $this->whenRootWorkspaceWasCreated($event),
-            NodeAggregateWasMoved::class => $this->whenNodeAggregateWasMoved($event),
+            NodeAggregateWasMoved::class => $this->whenNodeAggregateWasMoved($event, $eventEnvelope),
```

Comment:
```suggestion
            NodeAggregateWasMoved::class => $this->whenNodeAggregateWasMoved($event),
```

`$eventEnvelope` does not appear to be used.

Reply:
Yep, it used to, but no more